### DISCUSSION TITLE
minified tonal.js stopped working?

### DISCUSSION THREAD

**Discussion by jornswart** [tonaljs/tonal#359] [Q&A] [answered]
I made a simple chord creator app that worked fine for a year at https://pianostringtheory.com/wp-content/tools/chord-progression-creator/v3/chordprogressioncreatorv3.html but recently stopped working. I've tried to debug, and it appears that the minified tonal.js doesn't work anymore. The example in the documentation returns an error as well: 

<!DOCTYPE html>
<html> 
  <head>
    <script src="https://cdn.jsdelivr.net/npm/tonal/browser/tonal.min.js"></script>
    <script>
      console.log(Tonal.Key.minorKey("Ab"));
    </script>
    <title>test</title> 
  </head>

  <body> 
  </body>
</html>

Uncaught ReferenceError: Tonal is not defined

Has anyone else experienced this and found a solution?

**Comment by jlbocquet** [comment]
I encountered the same error on my site :
https://webaudiostudio.jlbee.fr
No workaround at this time ...

**Comment by jlbocquet** [comment]
I have found a old version of tonal.js in my archive.
Not sure that this old version implements all new features but it does not have the "Tonal not defined" regression !

Link to download :
https://webaudiostudio.jlbee.fr/js/lib/tonal.min.js

**Reply by jornswart** [reply]
Thanks so much! This works. I guess it's always a good idea to use a copy on your own website.

**Reply by zgaus11** [reply]
The link you provided is supposed to go in the source URL quotes correct?

**Comment by tschwarzl** [comment]
Dear all, 
unfortunately, I can confirm that the the tonal.min.js does not work:
https://jsfiddle.net/tschwarzl/c2u6amr7/18/#&togetherjs=tuJ0CnkRBQ

**Comment by danigb** [comment, marked as answer]
This was fixed a couple of days ago here: https://github.com/tonaljs/tonal/commit/0c66ef9029f23bf45075792cc4e03b0c89c30e58

Sorry for the inconvenience 🙏
